### PR TITLE
FLUID-6204: Updates for letter spacing preference

### PR DESCRIPTION
**Pull request by jobara**
https://issues.fluidproject.org/browse/FLUID-6204

### CODE REVIEW

**Comment by jobara** [comment]
@cindyli would you mind to review this PR. Seems i forgot to submit it when I made the Infusion change.

File: src/documents/APIChangesFrom2_0To3_0.md [hunk 1]
```diff
@@ -102,23 +99,205 @@ All panels must supply a `header` selector, which will be used by the `fluid.pre
...
+    </tbody>
+</table>
+
+##### Links and Buttons
+
+The "Links and Buttons" adjusters and enactors are collapsed to a single preference called "Enhance Inputs".
+
+###### Message Bundle Changes
+
+<table>
```
Comment (cindyli) [inline src/documents/APIChangesFrom2_0To3_0.md]: Are message bundle changes and model changes mixed up in this table? The change from "textSize" to "value" looks like a model change while file names are message bundles.

**Comment by jobara** [comment]
@cindyli thanks for catching that issue. I've updated the docs. This PR is ready for more review.

File: src/documents/APIChangesFrom2_0To3_0.md [hunk 2]
```diff
@@ -102,23 +99,223 @@ All panels must supply a `header` selector, which will be used by the `fluid.pre
...
+                    <li><code>inputsLarger</code></li>
+                <ul>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+###### Message Bundle Changes
+
+<table>
```
Comment (cindyli) [inline src/documents/APIChangesFrom2_0To3_0.md]: This table doesn't specify what messages have been changed in which json file.

File: src/documents/APIChangesFrom2_0To3_0.md [hunk 3]
```diff
@@ -102,23 +99,223 @@ All panels must supply a `header` selector, which will be used by the `fluid.pre
...
+            <td rowspan="4">tableOfContents.json</td>
             <td><code>"label"</code></td>
             <td><code>"tocLabel"</code></td>
         </tr>
         <tr>
             <td><code>"description"</code></td>
             <td><code>"tocDescr"</code></td>
         </tr>
+        <tr>
+            <td><code>""switchOn""</code></td>
```
Comment (cindyli) [inline src/documents/APIChangesFrom2_0To3_0.md]: Remove the extra `"`(double quotes). same at line 216 for "switchOn"

**Comment by jobara** [comment]
@cindyli i've addressed the issues you mentioned for the message bundles and the extra "". Ready for more review.

**Comment by cindyli** [comment]
Merged at 29afe5dadcf5bac11eaba7625c126790d16b87cb

**Pull request merged**